src/pages/index.js

```diff
@@ -153,7 +153,7 @@ function handleAvatarChange(data) {
       avatarPopup.renderLoading(false)
       userInfo.setUserInfo(data)
     })
-    .then(avatarPopup.close())
+    .then(avatarPopup.close)
     .finally(avatarPopup.renderLoading(true))
     .catch((err) => {
       console.log(console.error);
```

Comment:
```js

function handleProfileEditSubmit(data) {
  api.updateProfile(data.name, data.profession)
    .then((data) => {
      userInfo.setUserInfo(data)
    })
    .catch((err) => {
      console.log(console.error);
    });
  editProfilePopup.close();
}
```


You don't have `renderLoading` for the profile submit button 


` editProfilePopup.close();` should be in `then`


```js
    .then((data) => {
      userInfo.setUserInfo(data);
      editProfilePopup.close();  // here
    })

```